rclrs/build.rs

```diff
@@ -21,9 +22,8 @@ fn main() {
     println!("cargo:rustc-cfg=ros_distro=\"{ros_distro}\"");
 
     let mut builder = bindgen::Builder::default()
-        .header("src/rcl_wrapper.h")
+        .header(BINDGEN_WRAPPER)
         .derive_copy(false)
-        .allowlist_recursively(true)
```

Comment:
This is true by default ([source](https://docs.rs/bindgen/latest/src/bindgen/options/mod.rs.html#1372))